server/src/main/java/io/crate/expression/InputFactory.java

```diff
@@ -238,7 +240,13 @@ public Input<?> visitReference(Reference ref, Void context) {
             }
             implementation = referenceResolver.getImplementation(ref);
             if (implementation == null) {
-                throw new IllegalArgumentException("Column implementation not found for: " + ref);
+                String schema = ref.ident().tableIdent().schema();
+                String msg = SysSchemaInfo.NAME.equals(schema) ||
+                    InformationSchemaInfo.NAME.equals(schema) ?
+                    "Column: " + ref + " is missing from one or more of the cluster nodes. Instead of using 'SELECT *'" +
+                        "use a defined list of the columns to return" :
+                    "Column implementation not found for: \" + ref";
```

Comment:
```suggestion
                    "Column implementation not found for: " + ref + ". This can happen in mixed clusters when using `SELECT *`; Declare the column list explicitly instead" +
                    : "Column implementation not found for: \" + ref";
```